packages/libbpf/libbpf.spec

```
@@ -0,0 +1,51 @@
+Name: %{_cross_os}libbpf
+Version: 1.4.3
+Release: 1%{?dist}
+Summary: Library for BPF
+License: LGPL-2.1 OR BSD-2-Clause
```

`LGPL-2.1` is deprecated as an identifier:
```suggestion
License: LGPL-2.1-only OR BSD-2-Clause
```

---

ah thanks for this catch!